src/opm/output/eclipse/WriteRFT.cpp

```diff
@@ -599,6 +601,7 @@ namespace {
 
         virtual void addConnection(const ::Opm::UnitSystem&       usys,
                                    const ::Opm::Well&             well,
+                                   const double cell_depth,
```

Comment:
I don't think you really need the cell depths for the PLT records&ndash;be it for multi-segmented wells or otherwise&ndash;since that information is already in the RFT records.